### ISSUE TITLE
Specifying path with '/' incorrectly parses

### ISSUE THREAD

**Issue by mkwhitacre** [davidledwards/zookeeper#14]
When downloading and using the latest zookeeper-cli, when I try to specify a default path that contains multiple segments it fails stating that the path cannot end with the "/" character even though it does not.

```
$ ./zk node.example.com -p /kafka/wolfe
internal error: Path must not end with / character
>> stack trace
java.lang.IllegalArgumentException: Path must not end with / character
	at org.apache.zookeeper.common.PathUtils.validatePath(PathUtils.java:58)
	at org.apache.zookeeper.client.ConnectStringParser.<init>(ConnectStringParser.java:57)
	at org.apache.zookeeper.ZooKeeper.<init>(ZooKeeper.java:443)
	at com.loopfor.zookeeper.Zookeeper$.apply(Zookeeper.scala:737)
	at com.loopfor.zookeeper.Zookeeper$.apply(Zookeeper.scala:713)
	at com.loopfor.zookeeper.cli.CLI$.run(CLI.scala:103)
	at com.loopfor.zookeeper.cli.CLI$.main(CLI.scala:33)
	at com.loopfor.zookeeper.cli.CLI.main(CLI.scala)
```

**Comment by davidledwards** [comment]
The internal exception is being thrown because the `-p PATH` option appears after the server endpoint. Essentially, the underlying zookeeper library thinks you're specifying `-p` and `/kafka/wolfe` as servers in addition to `node.example.com`. Reordering the command line will solve your problem:
```
./zk -p /kafka/wolfe node.example.com
```
That said, I'll plan to add syntax validation of server endpoints so this problem is more apparent.

**Comment by mkwhitacre** [comment]
Ahh, my mistake on messing up the options coming before the host.

**Comment by davidledwards** [comment]
Validation of server host names will now occur before sending them to ZooKeeper, thereby avoiding the confusing internal error. Validation rules are defined [here](http://en.wikipedia.org/wiki/Hostname#Restrictions_on_valid_host_names).

Your example above will now result in the following:
```
$ zk node.example.com -p /kafka/wolfe
/kafka/wolfe: invalid host name
```
There is a possibility that erroneous placement of options could result in what appears to be well-formed host names. Therefore, I added a console message at startup to indicate how the CLI is connecting to the ZK quorum.

Example:
```
$ zk node.example.com -p kafka
connecting to {node.example.com:2181,-p:2181,kafka:2181} @ / ...
```
Here, is it obvious that the `-p` option has been positioned improperly.

To suppress messages, the `--quiet, -q` option has been added.
```
zk -q -p /kafka/wolfe foo.com bar.com
```
